app/src/main/java/ar/com/wolox/android/homepage/HomeActivity.kt

```diff
@@ -0,0 +1,24 @@
+package ar.com.wolox.android.homepage
+
+import android.content.Context
+import android.content.Intent
+import ar.com.wolox.android.R
+import ar.com.wolox.android.databinding.ActivityBaseBinding
+import ar.com.wolox.wolmo.core.activity.WolmoActivity
+
+class HomeActivity : WolmoActivity<ActivityBaseBinding>() {
+
+    override fun layout() = R.layout.home_activity
```

Comment:
It's ok by now. I don't think it has a point to create all the MVP boilerplate files just for these empty screen. But have in mind that, in a real scenario, you would have to create also the Fragment and the Presenter.